src/managers/GuildMemberManager.js

```diff
@@ -1,21 +1,32 @@
 'use strict';
 
-const DataStore = require('./DataStore');
+const BaseManager = require('./BaseManager');
 const GuildMember = require('../structures/GuildMember');
 const { Events, OPCodes } = require('../util/Constants');
 const Collection = require('../util/Collection');
 const { Error, TypeError } = require('../errors');
 
 /**
- * Stores guild members.
- * @extends {DataStore}
+ * Manages API methods for GuildMembers and stores their cache.
+ * @extends {BaseManager}
  */
-class GuildMemberStore extends DataStore {
+class GuildMemberManager extends BaseManager {
   constructor(guild, iterable) {
     super(guild.client, iterable, GuildMember);
+    /**
+     * The guild this manager belongs to.
```

Comment:
```suggestion
     * The guild this manager belongs to
```

Inconsistent capitalisation on `guild`